nntrainer/utils/util_func.cpp

```diff
@@ -92,21 +93,18 @@ Tensor zero_pad(int batch, Tensor const &in, unsigned int const *padding) {
 }
 
 // This is strip pad and return original tensor
-Tensor strip_pad(Tensor const &in, unsigned int const *padding,
-                 Tensor &output) {
+Tensor strip_pad(Tensor const &in, unsigned int const *padding, Tensor &output,
+                 unsigned int batch) {
   if (output.uninitialized()) {
-    output = Tensor(in.batch(), in.channel(), in.height() - padding[0] * 2,
+    output = Tensor(output.batch(), in.channel(), in.height() - padding[0] * 2,
```

Comment:
FYI, this will never happen. If output has batch it means it is initialized.
I guess this should be just using batch??

Reply:
done.